libs/designer-ui/src/lib/floatingactionmenu/floatingactionmenuinputs/helper.ts

```diff
@@ -65,7 +65,7 @@ export function deserialize(value: ValueSegment[], isRequestApiConnectionTrigger
 
   itemsProperties = rootObject;
   if (isRequestApiConnectionTrigger) {
-    itemsProperties = rootObject.properties ? rootObject.properties.rows.items : rootObject.rows.items;
+    itemsProperties = rootObject.properties?.rows?.items || rootObject.rows?.items || itemsProperties;
```

Comment:
Can you have a description on how to hit this issue previously?

Reply:
Done.